public/locales/en/app.json

```diff
@@ -68,6 +69,7 @@
     "peers": "Peers",
     "pinNoun": "Pin",
     "pins": "Pins",
+    "pinStatus": "Pin Status",
```

Comment:
nit: do we need this label at all? It looks bit noisy, and confusing when there are no pins.

Reply:
@lidel I'd say yes - eventually this will be a sortable field.

Reply:
Ah, indeed, sorting by pin status would be handy. 
Nvm then, we can refile label later (if needed).